docs/internal/config.md

```diff
@@ -14,6 +16,12 @@ This document lists advanced configuration settings rarely changed by users.
 |-------------------------------------------|--------------------------------------------------------------------------|---------------|
 | `OTEL_DOTNET_AUTO_DUMP_ILREWRITE_ENABLED` | Lets the profiler dump the IL original code and modification to the log. | `false`       |
 
+## Redirection
```

Comment:
Maybe "Managing dependencies", "Managing dependency conflicts" ? ... the "Redirection" alone here is not describing much the actual issue.

Reply:
btw, I think this was actually intentionally public per @pjanotti ?

Reply:
Maybe "Assembly Loading"?

Reply:
I will bring this line back to public. I thought it is more internal 😉

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2070/commits/f1c00c816ea116a9372ff0cf39542d73720ca5d7